qualang_tools/optimal_weights/TwoStateDiscriminator.py

```diff
@@ -8,8 +8,8 @@
 
 
 class TwoStateDiscriminator(StateDiscriminator):
-    def __init__(self, qmm, config, update_tof, rr_qe, path, meas_len, smearing, lsb):
-        super().__init__(qmm, config, update_tof, rr_qe, path, meas_len, smearing, lsb)
+    def __init__(self, qmm, config, update_tof, rr_qe, rr_pulse, path, meas_len, smearing, lsb):
+        super().__init__(qmm, config, update_tof, rr_qe, rr_pulse, path, meas_len, smearing, lsb)
```

Comment:
We also need to add docstring to this class and to the init function

Reply:
thanks